source/_docs/terminus/plugins/create.md

```diff
@@ -184,122 +184,141 @@ The first digit of a plugin version should always the Terminus MAJOR version the
-    "description": "A Hello, World Terminus command",
-    "type": "terminus-plugin",
-    "autoload": {
-        "psr-4": { "Pantheon\\TerminusHello\\": "src" }
-    },
-    "require": {
-        "organization/project-name": "^1"
-    },
-    "extra": {
-        "terminus": {
-            "compatible-version": "^1.1"
+1. Copy the `require-dev` and `scripts` sections from the composer.json file below into the `composer.json` of your Terminus plugin:
+
+    ```
+    {
+        "name": "my-username/terminus-hello-world",
+        "description": "A Hello, World Terminus command",
+        "type": "terminus-plugin",
+        "autoload": {
+            "psr-4": { "Pantheon\\TerminusHello\\": "src" }
+        },
+        "require": {
+            "organization/project-name": "^1"
+        },
+        "extra": {
+            "terminus": {
+                "compatible-version": "^1.1"
+            }
+        }
+        "require-dev": {
+            "squizlabs/php_codesniffer": "^2.7"
+        },
+        "scripts": {
+            "install-bats": "if [ ! -f bin/bats ] ; then git clone https://github.com/sstephenson/bats.git; mkdir -p bin; bats/install.sh .; fi",
+            "bats": "TERMINUS_PLUGINS_DIR=.. bin/bats tests",
+            "cs": "phpcs --standard=PSR2 -n src",
+            "cbf": "phpcbf --standard=PSR2 -n src",
+            "test": [
+                "@install-bats",
+                "@bats",
+                "@cs"
+            ]
         }
     }
-    "require-dev": {
-        "squizlabs/php_codesniffer": "^2.7"
-    },
-    "scripts": {
-        "install-bats": "if [ ! -f bin/bats ] ; then git clone https://github.com/sstephenson/bats.git; mkdir -p bin; bats/install.sh .; fi",
-        "bats": "TERMINUS_PLUGINS_DIR=.. bin/bats tests",
-        "cs": "phpcs --standard=PSR2 -n src",
-        "cbf": "phpcbf --standard=PSR2 -n src",
-        "test": [
-            "@install-bats",
-            "@bats",
-            "@cs"
-        ]
-    }
-}
-```
-Once you have done this, you can do the following:
+    ```
 
-- Run `composer install` to install the PHP Code Sniffer.
-- Run `composer cs` to check the coding standards of your plugin for PSR-2 compliance.
-- Run `composer cbf` to fix PRS-2 compliance errors in your plugin.
+2. Install the PHP Code Sniffer:
 
-The test scripts will create some working files, so you will probably want to add the following content to your plugin's `.gitignore` file:
-```bash
-vendor
-bats
-bin
-libexec
-share
-```
+        composer install
 
-Next, define some Bats tests to run. Create a folder named `tests`, and create a file named `confirm-install.bats`. Put the contents below in your Bats test file.
+3. Check the coding standards of your plugin for PSR-2 compliance:
 
-```
-#!/usr/bin/env bats
+        composer cs
 
-#
-# confirm-install.bats
-#
-# Ensure that Terminus and the Composer plugin have been installed correctly
-#
+4. Fix any PRS-2 compliance errors in your plugin:
```

Comment:
`cbf` fixes most problems, but there are some that it just tells you about and does not fix. For example, if your method is missing a `public`, `protected` or `private`, it doesn't know which was intended, so it tells you to add  the visibility modifier yourself. Maybe just changing `any` to `most` is sufficient.